### DISCUSSION TITLE
AVIF images are pixelated

### DISCUSSION THREAD

**Discussion by maksg** [ente-io/ente#2024] [Q&A]
Currently my backup gets stuck with uploading 2 memories on iOS. I don't know how to check which ones but the only 2 AVIF images I have appear pixelated on my Android device so I think it's safe to assume that it's these ones.
I'm using self-hosted instance on Docker and haven't tested on your cloud service so I don't know if it works there.

I put the AVIF in zip because the GitHub doesn't let me upload it otherwise.

![Screenshot_20240606-001118](https://github.com/ente-io/ente/assets/11507974/7c1c5e11-eaf8-4a66-a238-55530f50fffd)
![IMG_2208](https://github.com/ente-io/ente/assets/11507974/ae88b970-df59-4c76-bf8b-7b2cc1680a4c)
[IMG_8167.zip](https://github.com/user-attachments/files/15597329/IMG_8167.zip)

**Comment by maksg** [comment]
I just tried creating new AVIF images and they uploaded right away so stuck backup seems to be a separate issue. But they are also pixelated on Android so there might be something wrong with them anyway.

**Comment by gabriele2000** [comment]
I suggest to create an issue for it.
I also have this problem and this is not really the place to have this fixed... and we need it fixed (AVIF is the most efficient image format)

**Comment by ua741** [comment]
https://github.com/flutter/flutter/issues/61229
